lib/SILOptimizer/Analysis/ArraySemantic.cpp

```diff
@@ -682,3 +684,42 @@ bool swift::ArraySemanticsCall::replaceByValue(SILValue V) {
   removeCall();
   return true;
 }
+
+bool swift::ArraySemanticsCall::replaceByAppendingValues(SILModule &M,
+  SILFunction *AppendFn,
+  SmallVectorImpl<SILValue> &Vs,
+  ArrayRef<Substitution> Subs) {
+  assert(getKind() == ArrayCallKind::kAppendContentsOf &&
```

Comment:
The formatting here (and in general inside your patches) is a bit strange. Could you try to use clang-format. Actually, you can use git-clang-format (which you can find in the `clang/tools/clang-format/git-clang-format`  source directory. This script will only format your changes (compared to HEAD). We use the LLVM style with clang-format.